pvp_api_doc_exemptions.json

```diff
@@ -0,0 +1,29 @@
+{
+  "exempts": {
+    "PVP-20-1": {
+      "errors": [
+        "Missing Doc on System.String Cinemachine.Editor.CinemachineTool::GetIconPath()",
+        "Missing Doc on System.Void Cinemachine.CinemachineBlendListCamera::Reset()",
+        "Missing Doc on System.Void Cinemachine.CinemachineClearShot::Reset()",
+        "Missing Doc on System.Void Cinemachine.CinemachineMixingCamera::Reset()",
+        "Missing Doc on Cinemachine.CinemachineStoryboard",
```

Comment:
What errors does his line fix?

Reply:
You mean line 9?  The PVP (package verification profile) API doc validation test reports `Cinemachine.CinemachineStoryboard` has no doc. But I checked in the source codes, class `CinemachineStoryboard` has docs as expected. 
This could be a bug in PVP itself. Maybe it cannot handle a class with several attributes.  That's why I exempt this error for now.

Reply:
I see.  I checked the code, and the reason that CinemachineStoryboard reports as undocumented is because it's defined inside a `#if CINEMACHINE_UGUI` block.  Apparently the doc tools don't define these things, so XML documentation for a number of our classes will be missing.